theseus/embodied/measurements/tests/test_between.py

```diff
@@ -125,3 +124,30 @@ def test_error_between_se2():
                 error = dist_cf.error()
                 assert np.allclose(error.squeeze().numpy(), between_errors[z])
                 z += 1
+
+
+def test_jacobian_between_se3():
+    for batch_size in [1, 20, 100]:
+        aux_id = torch.arange(batch_size)
+        se3_1 = th.SE3.rand(batch_size, dtype=torch.float64)
+        se3_2 = th.SE3.rand(batch_size, dtype=torch.float64)
+        measurement = th.SE3.rand(batch_size, dtype=torch.float64)
+        dist_cf = th.eb.Between(
+            se3_1, se3_2, th.ScaleCostWeight(1.0), measurement=measurement
+        )
+
+        actual = dist_cf.jacobians()[0]
+
+        def test_fun(g1, g2):
+            dist_cf.v0.update(g1)
+            dist_cf.v1.update(g2)
+            return dist_cf.error()
+
+        jac_raw = torch.autograd.functional.jacobian(test_fun, (se3_1.data, se3_2.data))
```

Comment:
Does it make sense to also check against the numerical/finite difference version?

Reply:
They are tested on `SE2`. I can add `SE3` as well.

Reply:
Numerical `jacobian` tests are added.